deeppavlov/models/elmo/elmo.py

```diff
@@ -272,6 +273,9 @@ def __init__(self,
         self._build_model(train=False, epoch=load_epoch_num)
 
         self.save()
+        # after building the model and saving to the specified save path
+        # you need to change the way to load the checkpoint
```

Comment:
`you need to change` sounds like a requirement for a user to do something
```suggestion
        # change the way to load intermediate checkpoints
```